sample-input/sph-factors/assembly/sph-region-avg.py

```diff
@@ -0,0 +1,119 @@
+import matplotlib.pyplot as plt
```

Comment:
Have you run into any problems running this input file on clusters? I'm running it on AWS and getting the error shown below. I'm guessing it has to do with matplotlib no longer having a headless backend or something. 

  File "sph-region-avg.py", line 91, in <module>
    openmoc.plotter.plot_materials(openmoc_geometry)
  File "/home/shaner/.local/lib/python2.7/site-packages/openmoc/plotter.py", line 238, in plot_materials
    plot_spatial_data(materials, plot_params)
  File "/home/shaner/.local/lib/python2.7/site-packages/openmoc/plotter.py", line 919, in plot_spatial_data
    fig = plt.figure()
  File "/usr/lib/pymodules/python2.7/matplotlib/pyplot.py", line 423, in figure
    **kwargs)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 79, in new_figure_manager
    return new_figure_manager_given_figure(num, figure)
  File "/usr/lib/pymodules/python2.7/matplotlib/backends/backend_tkagg.py", line 87, in new_figure_manager_given_figure
    window = Tk.Tk()
  File "/usr/lib/python2.7/lib-tk/Tkinter.py", line 1767, in __init__
    self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
_tkinter.TclError: no display name and no $DISPLAY environment variable